cmd/rundir.go

```diff
@@ -13,19 +12,19 @@ func rundir(dirPath string) {
 	if err != nil {
 		panic(err)
 	}
+	log := logger.New(c.LogLevel)
 
 	reader := recipe.NewReader()
 	recipeList, err := reader.ReadDir(dirPath)
 	if err != nil {
 		panic(err)
 	}
-	runner, cleanFn := initRunner(c)
+	runner, cleanFn := initRunner(c, log)
 	defer cleanFn()
 	faileds, err := runner.RunMultiple(recipeList)
```

Comment:
```faileds``` can be renamed to ```failed_recipes``` or ```failures```, wdyt??

Reply:
yup definitely can be changed.
how about to `failedRecipes` for following convention?

Reply:
Yup seems better